streamlit_app/app.py

```
@@ -0,0 +1,259 @@
+        "_result_state": None,
+        "result": None,
+        "run_error": None,
+        "dataset_path": None,
+        "dataset_has_gold": False,
+    }
+    for key, val in defaults.items():
+        if key not in st.session_state:
+            st.session_state[key] = val
+
+
+def _start_run(dataset_path: Path, model: str):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_start_run`

Add return type annotation: `None`

(ANN202)